packages/edge-login-ui-rn/src/reducers/RootReducer.ts

```diff
@@ -10,15 +10,15 @@ import {
 } from './PasswordRecoveryReducer'
 import { passwordStatus, PasswordStatusState } from './PasswordStatusReducer'
 import { previousUsers, PreviousUsersState } from './PreviousUsersReducer'
-import { workflow, WorkflowState } from './WorkflowReducer'
+import { scene, SceneState } from './WorkflowReducer'
```

Comment:
Since it's `scene` and `SceneState`, I think the file should be renamed to `SceneReducer`. Please do this in its own commit, after the current one. If you rename a file and make changes at the same time, Git sometimes gets confused, so we typically do renames separately.